### DISCUSSION TITLE
Compiling error in ubuntu 20.04

### DISCUSSION THREAD

**Discussion by mailonghua** [microsoft/msquic#1352] [Q&A] [answered]
cmake version: cmake version 3.20.0-rc5
1.Run "./scripts/prepare-machine.ps1 -Configuration Dev " and output warning:
`WARNING: Microsoft.Logging.CLOG could not be installed. Building with logs will not work
WARNING: The remote certificate is invalid because of errors in the certificate chain: PartialChain
WARNING: Microsoft.Logging.CLOG2Text.Lttng could not be installed. Building with logs will not work
WARNING: The remote certificate is invalid because of errors in the certificate chain: PartialChain
`
**2.run /scripts/build.ps1 and output Err msg:**
`PS /home/mailonghua/project/msquic/src/msquic> ./scripts/build.ps1
[03/19/2021 01:32:55] Generating files...
[03/19/2021 01:32:55] cmake -g 'Ninja' -DQUIC_TLS=openssl -DQUIC_OUTPUT_DIR=/home/mailonghua/project/msquic/src/msquic/artifacts/bin/linux/x64_Debug_openssl -DQUIC_ENABLE_LOGGING=on -DCMAKE_BUILD_TYPE=Debug ../../..
CMake Error: Unknown argument -g
CMake Error: Run 'cmake --help' for all supported options.
Write-Error: /home/mailonghua/project/msquic/src/msquic/scripts/build.ps1:320
Line |
 320 |      CMake-Execute $Arguments
     |      ~~~~~~~~~~~~~~~~~~~~~~~~
     | [03/19/2021 01:32:55] CMake exited with status code 1
`

**Comment by nibanks** [comment, marked as answer]
Did you restart your terminal after installing .NET Core?
